features/admin/product/managing_product_options/managing_values_of_product_option.feature

```diff
@@ -11,15 +11,15 @@ Feature: Managing option values of a product option
         And this product option has also the "M" option value with code "OV2"
         And I am logged in as an administrator
 
-    @todo @ui @mink:chromedriver @api
+    @ui @mink:chromedriver @api
     Scenario: Adding an option value to an existing product option
         When I want to modify the "T-Shirt size" product option
         And I add the "L" option value identified by "OV3"
         And I save my changes
         Then I should be notified that it has been successfully edited
         And this product option should have the "L" option value
 
-    @todo @ui @javascript @todo
+    @ui @javascript @todo
```

Comment:
```suggestion
    @ui @javascript
```
No idea why there were 2

Reply:
I'm aware of that, we havent got such a "feature" like removing the values https://github.com/Sylius/Sylius/pull/6799. After I consumed some out of scope time on other things around this task I just wanted to bring it the same as it was